thirdeye-ui/src/app/pages/subscription-groups-create-page/subscription-groups-create-page.component.tsx

```diff
@@ -38,14 +41,22 @@ import {
 import { createEmptySubscriptionGroup } from "../../utils/subscription-groups/subscription-groups.util";
 
 export const SubscriptionGroupsCreatePage: FunctionComponent = () => {
-    const [loading, setLoading] = useState(true);
-    const [alerts, setAlerts] = useState<Alert[]>([]);
     const navigate = useNavigate();
     const { t } = useTranslation();
     const { notify } = useNotificationProviderV1();
 
+    const { alerts, getAlerts, status: alertsStatus } = useGetAlerts();
+    const {
+        enumerationItems,
+        getEnumerationItems,
```

Comment:
Does this fetch all the enumeration items that exist in the table?

Reply:
Yes. The API call doesn't take a lot longer than a call for specific IDs, so the create/update pages fetch all the items to show them all as options, without a new call each time a new alert is selected.

Reply:
I see. This is a performance issue waiting to happen as more enumeration items are generated but I'll let this slide this time. The API can filter by alert id(s) now right?

Reply:
Yes, the list API now support filtering by alert id (don't think it supports multiple, but probably don't need that in this case).

Should I add a ticket to upgrade this API to make alert ID specific API call for each alert selection?

Reply:
Yes let's create a ticket for that. We probbaly will need to refactor so those are on a need to have basis

Reply:
Created a ticket: [TE-1360](https://cortexdata.atlassian.net/browse/TE-1270)